src/main/java/com/mitchej123/hodgepodge/LoadingConfig.java

```diff
@@ -200,6 +201,7 @@ public LoadingConfig(File file) {
         arabicNumbersForEnchantsPotions = config.get(Category.TWEAKS.toString(), "arabicNumbersForEnchantsPotions", false, "Uses arabic numbers for enchantment levels and potion amplifier levels instead of roman numbers").getBoolean();
         bedMessageAboveHotbar = config.get(Category.TWEAKS.toString(), "bedMessageAboveHotbar", true, "Show \"cannot sleep\" messages above hotbar").getBoolean();
         chatLength = Math.max(100, Math.min(32767, config.get(Category.TWEAKS.toString(), "chatLength", 8191, "Amount of chat lines kept [100(Vanilla) - 32767]").getInt()));
+        messageLength = Math.max(100, Math.min(512, config.get(Category.TWEAKS.toString(), "messageLength", 256, "Length of messages sent in chat [100(Vanilla) - 512").getInt()));
```

Comment:
can you remove this setting and just hardcode it to 256 please

Reply:
Or sync the value from server to client

Reply:
> Or sync the value from server to client

that's too much effort for such a feature